tasks/rtloader.py

```diff
@@ -79,3 +79,50 @@ def format(ctx, raise_if_changed=False):
             for f in changed_files:
                 print("  - {}".format(f))
             raise Exit(code=1)
+
+@task
+def generate_doc(ctx):
+    """
+    Generates the doxygen documentation, puts it in rtloader/doc, and logs doc errors/warnings.
+    (rtloader/doc is hardcoded right now in the Doxyfile, as doxygen cannot take the output directory as argument)
+    Logs all errors and warnings to <rtloader_path>/doxygen/errors.log and to the standard output.
+    Returns 1 if errors were found (by default, doxygen returns 0 even if errors are present).
+    """
+    rtloader_path = get_rtloader_path()
+
+    # doxygen puts both errors and warnings in stderr
+    result = ctx.run("doxygen {0}/doxygen/Doxyfile 2>{0}/doxygen/errors.log".format(rtloader_path), warn=True)
+
+    if result.exited != 0:
+        print("Fatal error encountered while trying to generate documentation.")
+        print("See {0}/doxygen/errors.log for details.".format(rtloader_path))
```

Comment:
The error looks like: `See C:\XXX\datadog-agent\tasks\..\rtloader/doxygen/errors.log for details.`. You might normalize `rtloader_path`.